deploy/crds/wildfly.org_wildflyservers_crd.yaml

```diff
@@ -210,6 +210,33 @@ spec:
                 format: int32
                 minimum: 0
                 type: integer
+              resources:
+                description: 'Configuration of the resources used by the WildFlyServer,
```

Comment:
Minor detail, the documentation of this field in the CRD is different than the one expressed by the WildFlySererType https://github.com/wildfly/wildfly-operator/pull/215/files#diff-7ded448f02c4df76f693b89f0f2c52359f34cf3a0bbd4aa04517227f956c5065R51

Could you generate the CRD again?

Reply:
Thanks, I've regenerated the CRD so that it's in sync with the Go types